src/main/jbhook1/avs-boot.c

```diff
@@ -95,14 +97,44 @@ static void my_avs_boot(
 {
     log_info("Called my_avs_boot");
 
-    avs_boot_replace_property_uint32(config, "log/level", 4);
-    avs_boot_replace_property_str(
+    // avshelper gates this behind a check that might not always be true
+    CreateDirectoryA("./CONF", 0);
+    CreateDirectoryA("./CONF/NVRAM", 0);
+    CreateDirectoryA("./CONF/RAW", 0);
```

Comment:
Got a few questions here:

- Relative path to current working dir, is that fine? If yes, why?
- Why are you using `CreateDirectoryA` and not the AVS filesystem functions? Using the AVS functions, I am wondering if this might benefit from the whole `avs-config` thing where you can configure your file systems and paths.

Remark: I have no idea how this works on jubeat and what the game expects.

Reply:
Truthfully, no reason apart from "`avshelper.dll` does it, so I will do the same". If `F:` doesn't exist, that's exactly what the game does. We could instead match the other games and shift this dir creation out to the .bat files. Thoughts?

Reply:
Well, sticking to what the game does is probably not a bad default if you cannot see any issues with that. I guess we can also iterate that when needed. The current solution sounds reasonable as it also avoids complexity. Let's keep it like that for now.